src/components/common/Stars/StairsContainer.js

```diff
@@ -0,0 +1,15 @@
+import { connect } from 'react-redux';
+import {
+  createAction_changeMyStarsChoice,
+  createAction_delateMyStarsChoice,
+} from '../../../redux/productsRedux';
+import Stars from './Stars';
+
+const mapDispatchToProps = dispatch => ({
```

Comment:
Stairs jest ogólnym komponentem, może być zastosowany w wielu różnych miejscach, np. do oceny komentarzy. Tego typy komponenty nie powinny odpalać żadnych akcji redux-a, bo jest to logika ściśle połączona z daną logiką biznesową i uniemożliwia reużycie komponentu. Komponent `Staer` powinien w propsach przyjmować funkcję `onChangeStar(value)`, a co ma się stać po takiej zmianie powinno być zdefiniowane w kontenerze produktu, uwaga na przyszłość, nie musisz tego porpawiać